modules/op-release-notes-1-9.adoc

```diff
@@ -373,3 +373,42 @@ spec:
 * Before this update, a pull request failed if the user in the annotation provided values by using a regex form, for example, `refs/head/rel-*`. The pull request failed because it was missing `refs/heads` in its base branch. With this update, the prefix is added and checked that it matches. This resolves the issue and the pull request is successful.	
 // Savita Ashture
 
+[id="release-notes-1-9-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.9.1
+
+With this update, {pipelines-title} General Availability (GA) 1.9.1 is available on {product-title} 4.11 and later.
+
+[id="fixed-issues-1-9-1_{context}"]	
+== Fixed issues	
+
+* Before this update, the `tkn pac repo list` command did not run on Microsoft Windows. With this update, this issue is resolved and now you can run the `tkn pac repo list` command on on Microsoft Windows.
+
+* Before this update, {pac} watcher did not receive all of the configuration change events. With this update, this issue is resolved. The {pac} watcher is updated and does not miss the configuration change events.
+
+* Before this update, TaskRuns or PipelineRuns pods created by {pac} could not access custom certificates exposed by the user in the cluster. With this update, this issue has been resolved. You can now access custom certificates in the cluster from TaskRuns or PipelineRuns pods.
+
+* Before this update, the core interceptor `tekton-triggers-core-interceptors` used in the `Trigger` resource did not function after the Pipelines operator was upgraded to version 1.9. With this update, this issue is resolved. The core interceptor, `tekton-triggers-core-interceptors`, has been updated to TLS version 1.2, which ensures that the functionality of the core interceptor runs accurately.
```

Comment:
How about changing to something like this because I want to mention that this issue occurred on a Openshift cluster where FIPS enabled
Please feel free to modify sentence but make sure to mention FIPS
```suggestion
* Before this update, the core interceptor `tekton-triggers-core-interceptors` used in the `Trigger` resource did not function after the Pipelines operator was upgraded to version 1.9 on a cluster where FIPS enabled. With this update, this issue is resolved. The core interceptor, `tekton-triggers-core-interceptors`, has been updated to TLS version 1.2 (because OpenShift uses MInTLS as 1.2 for all components), which ensures that the functionality of the core interceptor runs accurately.
```

Reply:
Fixed.